### PR TITLE
Fixes for MinGW32 (gcc 4.8.1)

### PR DESCRIPTION
**Pull request by rioki**
Building under MinGW32 some minor things did not work. There changes address the issues.

### CODE REVIEW

**Comment by bnoordhuis** [comment]
Couple of style comments but apart from that no objections from me.

@piscisaureus PTAL.

**Comment by rioki** [comment]
Will fix the outstanding issues soon.

**Comment by rioki** [comment]
Fixed outstanding issues. 

The last commit about swprintf was removed, mingw-w64 has the proper version. Why mingw32 "lost" the standard conforming swprintf  in the latest version is beyond me. Assuming that the standard conforming version exists is the safer route. The problem with mingw32 vs. mingw-w64 is that you can not distinguish between the two.  

Also, rebased changes against joyen/libuv/master.

File: Makefile.am [hunk 1]
```diff
@@ -197,6 +195,16 @@ test_run_tests_SOURCES = test/blackhole-server.c \
                          test/test-walk-handles.c
 test_run_tests_LDADD = libuv.la
 
+if WINNT
+test_run_tests_SOURCES += test/runner-win.c \
+                          test/runner-wind.h
```
Comment (bnoordhuis) [inline Makefile.am]: Typo?

Comment (rioki) [inline Makefile.am]: No old change, only without whitespace change, or am I missing something? All changes are rebased, so the history looks nice.

Comment (bnoordhuis) [inline Makefile.am]: I mean that it's test/runner-win.h, not test/runner-wind.h. :-)

**Comment by rioki** [comment]
Fixes typo in Makefile.am, rebase from joyen/libuv/master.

**Comment by bnoordhuis** [comment]
Thanks, LGTM.  Can I ask you to sign [the CLA](http://nodejs.org/cla.html)?

I don't think we'll wait for @piscisaureus's review if we want to see this land this side of the Singularity. :-)

**Comment by rioki** [comment]
singed, real name is Sean Farrell (if that was not obvious from the commit...)

**Comment by bnoordhuis** [comment]
Cheers Sean, landed in ee434b3, 4c6294e and 63cd99e.

**Pull request closed without merging**